packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -18,35 +26,19 @@ const WorkflowStepStore = types
       return []
     },
 
-    isThereANextStep () {
-      const nextStep = self.getNextStepKey()
-      return nextStep && nextStep !== 'summary'
-    },
-
-    isThereAPreviousStep () {
-      const validStepReference = isValidReference(() => self.active)
-      if (validStepReference) {
-        const firstStep = self.steps.keys().next()
-        return self.active.stepKey !== 'summary' && self.active.stepKey !== firstStep.value
-      }
-
-      return false
-    },
-
     get isThereTaskHelp () {
       const tasks = self.activeStepTasks
 
       return tasks.some(task => task.help)
     },
 
     get shouldWeShowDoneAndTalkButton () {
-      const isThereANextStep = self.isThereANextStep()
       const workflow = tryReference(() => getRoot(self).workflows?.active)
       const classification = tryReference(() => getRoot(self).classifications?.active)
 
       if (workflow && classification) {
         const disableTalk = classification.metadata.subject_flagged
-        return !isThereANextStep &&
+        return !self.active.isThereANextStep &&
```

Comment:
Just for safety, let's check for a valid reference. See above comment for the addition.

```suggestion
        return step.isThereANextStep &&
```